### PR TITLE
Redis 44 create docker deployment

### PR DESCRIPTION
**Pull request by EngineersBox**
### Changelog

* Added Ansible workflow
* Added Ansible playbook for deploying services
* Fixed SSL issues
* Merged in @RongXin code for rate limiter
* Added missing tests for resolver in KeyedRateLimiter module

### CODE REVIEW

File: .github/workflows/ansible.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        env:
+          SSH_PRIVATE_KEY: ${{ secrets.DEPLOY_KEY }}
+      - name: Verify targets are accessible
+        run: sudo ansible all --private-key=$KEYS_DIR/$PRIVATE_KEY_FILE -m ping
+        working-directory: $ANSIBLE_DIR
+      - name: Run playbook
+        run: sudo ansible-playbook -v --private-key=$KEYS_DIR/$PRIVATE_KEY_FILE --extra-vars "ansible_sudo_pass=${{ secrets.SUDO_PASS }}" $PLAYBOOK
+        working-directory: $ANSIBLE_DIR
+      - name: Remove key file
+        run: sudo rm -rf $KEYS_DIR
```
Comment (ariadev79) [inline .github/workflows/ansible.yml]: Not familiar with ansible, but a new line needed?

**Review by ariadev79 — CHANGES_REQUESTED** [review]
Overall looks good!

- What's the `flask_logging.conf` for?
- Remove pyc and .idea
- Bunch of endline warnings

**Comment by EngineersBox** [comment]
the `flask_logging.conf` is used to create loggers for the flask applet, supplying timestamped and threaded entries as well as splitting between stdout and a file.

I'll remove the `pyc` and `.idea` content and fix the endlines

**Comment by EngineersBox** [comment]
@WeiXinFam I have addressed your comments, have a look when you get a sec

**Review by ariadev79 — APPROVED** [review]

**Pull request merged**